README.md

```diff
@@ -983,6 +983,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Harshit Parwal](https://github.com/harshitparwal)
   - [Harsit Agarwalla](https://github.com/harsitagarwalla187)
   - [Hartato Cipta Jaya](https://github.com/DHCJS)
+  - [Harshvardhan singh](https://github.com/HarshSingh-hue)
```

Comment:
Please add your name in line 984, after Harshit Parwal.
Let us know if you need any help with that. 🙂